android/history.md

```diff
@@ -10,15 +10,42 @@
 * Fix crashes from invalid package name/version (#819)
 * Clean up console log (#748)
 
+## 2018-04-30 10.0.385 beta
+* No changes to Keyman for Android (updated Keyman Web Engine, #834)
+
+## 2018-04-30 10.0.384 beta
+* No changes to Keyman for Android.
+
+## 2018-05-03 10.0.383 beta
+* No changes to Keyman for Android.
+
+## 2018-04-30 10.0.382 beta
+* No changes to Keyman for Android (updated Keyman Web Engine, #797)
+
+## 2018-04-30 10.0.381 beta
+* Fix OSK missing some keys on older Android configurations (#781)
```

Comment:
I thought we're including the issue number in history.md

Issue #304 was fixed in PR #781

Reply:
From team discussion, going forward, we're using PR numbers

Reply:
Per discussion in standup, we'll start using PR numbers for simplicity of tracking. But no need to change earlier issue numbers. Remember that visiting a PR, e.g. 123 at `https://github.com/keymanapp/keyman/issues/123` will redirect to the correct PR page anyway.